lib/TimeSeries/AdaptiveFilter.pm

```diff
@@ -0,0 +1,210 @@
+package TimeSeries::AdaptiveFilter;
+
+=head1 NAME
+
+TimeSeries::AdaptiveFilter - Adaptive filter for data stream with possible outliers
+
+=cut
+
+our $VERSION = '0.02';
+
+=head1 VERSION
+
+Version 0.02
+
+=head1 STATUS
+
+=begin HTML
+
+<p>
+    <a href="https://travis-ci.org/binary-com/perl-TimeSeries-AdaptiveFilter"><img src="https://travis-ci.org/binary-com/perl-TimeSeries-AdaptiveFilter.svg" /></a>
+</p>
+
+=end HTML
+
+=cut
+
+use strict;
+use warnings;
+
+use List::Util qw(sum max);
+
+use parent qw/Exporter/;
```

Comment:
Well, it's standard way for exporting function: inherit `Exporter` + define `@EXPORT_OK`. I'm fine with that :) The `import` should be used for some non-standard ways, i.e. when something should be generated on fly, or additional parameters are taken etc.